src/modules/jcms_notifications/src/MysqlNotificationStorage.php

```diff
@@ -35,8 +36,13 @@ public function __construct(Connection $connection) {
   /**
    * @inheritdoc
    */
-  public function saveNotificationNid(int $nodeId) {
-    return $this->connection->insert(self::TABLE)->fields(['node_id'], [$nodeId])->execute();
+  public function saveNotificationNid(EntityInterface $entity) {
```

Comment:
No, as it still saves the `nid`. The name is based on what it does rather than what it receives.